### PR TITLE
A10

### PR DESCRIPTION
**Pull request by allopez7**

### CODE REVIEW

File: solutions/10.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const callFunctionBigger = (funct, num1, num2, i=1)=>{
+  if(i > num1 && i > num2){
```
Comment (seemcat) [inline solutions/10.js:2]: if (i >num1 && num2)

File: test/10.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+const solution = require('../solutions/10.js');
+
+const biggerFunction = (numOne, numTwo)=>{
+  let i = 0;
+  let passing = true;
+  let bigger = 0;
+
+  if(numOne > bigger){
+    bigger = numOne;
+  }
```
Comment (seemcat) [inline test/10.js]: } else {
bigger = numTwo;
}

**Review by seemcat — CHANGES_REQUESTED** [review]

**Review by bramakrishna16 — APPROVED** [review]

File: solutions/10.js (hunk 1 above)
Comment (allopez7) [reply solutions/10.js:2]: this does not work for me.

**Review by allopez7 — COMMENTED** [review]
changes made!

**Pull request merged**